gordo_components/server/utils.py

```diff
@@ -353,9 +355,22 @@ def load_metadata(directory: str, name: str) -> dict:
     Returns
     -------
     dict
+    """
+    compressed_metadata = _load_compressed_metadata(directory, name)
+    return pickle.loads(zlib.decompress(compressed_metadata))
+
+
+@lru_cache(maxsize=25000)
+def _load_compressed_metadata(directory: str, name: str):
+    """Loads the metadata and returns it as a lzma compressed pickle, to use
```

Comment:
Very small point, most of the time we have a return under the `"""`  but the main thing is if we want to do our numpy docs here?

Reply:
Tja, IDK. Since it is an hidden function I think it is less important to have a full documentation, but it is nice to follow the same style as the rest, so I tried to make it numpy compliant.